src/main/java/taxi/controller/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,29 @@
+package taxi.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+
+@WebServlet("/myCar")
```

Comment:
Greate job! Just a few comments) maybe it's better to name your url as drivers/cars according to checklist